aea/crypto/base.py

```diff
@@ -99,16 +106,6 @@ def recover_message(
         :return: the recovered addresses
         """
 
-    @classmethod
-    @abstractmethod
-    def load(cls, fp: BinaryIO) -> "Crypto":
```

Comment:
Notice: we do the loading in the constructor, if `private_key_path is None`.

Thoughts? O/w we have to change the constructor so to accept an already loaded sequence of bytes, aka private key.